Module-1-Github/Day-4/Task/Hamzah.task

```diff
@@ -0,0 +1,20 @@
+
+1.Git rebase
+This document will serve as an in-depth discussion of the git rebase command. The Rebase command has also been looked at on the setting up a repository and rewriting history pages. This page will take a more detailed look at git rebase configuration and execution. Common Rebase use cases and pitfalls will be covered here.
+
+2.Git merge
```

Comment:
Try to understand and summarize the command why we need it.Try to merge or reset and cherry pick. Add steps to follow with commands so that you will get how things work.

Reply:
Command are added for Git merge ,Git reset , Git pull ,Git fetch , cherry pick